project_euler.net/tests.py

```diff
@@ -0,0 +1,31 @@
+class test:
+	def __init__(self,pvrple,blvck):
+		self.pvrple = pvrple
+		self.blvck = blvck
+
+	def test(pvrpleblvck):
+		pass
+
+
+# git branch <new-branch-name>
+# git branch -m <new-name>
+# git branch -m <old-name> <new-name>
+
+# First, delete the current / old branch:
+# git push origin --delete <old-name>
+
+# Then, simply push the new local branch with the correct name:
+# git push -u origin <new-name>
+
+# git checkout <other-branch>
+# git switch <other-branch>
+# git push -u origin <local-branch>
+# git branch --track <new-branch> origin/<base-branch>
+
+# (1) Check out the branch that should receive the changes
+# git switch main
+	
+# (2) Execute the "merge" command with the name of the branch that contains the desired changes
+# git merge feature/contact-form
+
+print('H')
```

Comment:
It would be good to have a specific unit test per Euler problem!

For a good read on testing with Python, this seems to be a good article: [A simple introduction to Test Driven Development with Python](https://www.freecodecamp.org/news/learning-to-test-with-python-997ace2d8abe/).

Reply:
Ok, I will keep that in mind. I did use ` self.AssertEqual` and `self.AssertTrue` in problem set four. The others were returning false. Mainly because they work most on strings or a different kind of function.

Reply:
Ok i see what you mean. Something like this


"""import unittest
from mycode import *

class MyFirstTests(unittest.TestCase):

def test_hello(self):
        self.assertEqual(hello_world(), 'hello world')

def test_custom_num_list(self):
        self.assertEqual(len(create_num_list(10)), 10)

    def test_custom_func_x(self):
        self.assertEqual(custom_func_x(3,2,3), 54) """

Reply:
Correct!